cos_alerter/config-defaults.yaml

```diff
@@ -9,11 +9,16 @@ watch:
   wait_for_first_connection: true
 
   # The list of Alertmanager instances we are monitoring. Alertmanager instances should be
-  # configured with the clientid=<client> parameter.
+  # configured with the clientid=<clientid> &key=<clientkey> parameters.
+  # <clientid> must be a valid UUID
```

Comment:
Why does the clientid need to be a UUID? It seems like it would make configuration a lot more annoying for humans while offering no benefit. As long as the clientids are unique from each other (something we should have been checking for before but I missed), there should be no issues.